Examples/Image/GettingStarted/07_Deconvolution_Visualizer.py

```diff
@@ -11,7 +11,7 @@
 from cntk.ops import combine
 from cntk.io import MinibatchSource, CTFDeserializer, StreamDef, StreamDefs
 from PIL import Image
-from cntk import graph
+from cntk.logging import graph
```

Comment:
The deconv example has been fixed and a test was added, the change is in master since a day or two. Please rebase your branch onto the latest master, your changes to the deconv example files should be obsolete by now. Thanks though.